code-of-conduct.md

```diff
@@ -1,113 +1,78 @@
-## Diversity Statement
+## Code of Conduct
+
+The Code of Conduct is designed to protect the US-RSE community and its members from physical and psychological harm.
+To this end, we consider the following behaviors unacceptable in any US-RSE community spaces, in person or online:
```

Comment:
I wonder if this list should be stated as a set of examples, rather than an exhaustive list, which it could be read as.